websites/N/Nexon Logistics/presence.ts

```diff
@@ -0,0 +1,180 @@
+const presence = new Presence({
+		clientId: "1046638491435552819",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	let presenceData: PresenceData = {
+		largeImageKey: "https://i.imgur.com/m740VV3.jpg",
+	};
+	const [buttons, timestamp, privacy] = await Promise.all([
+			presence.getSetting<boolean>("buttons"),
+			presence.getSetting<boolean>("timestamp"),
+			presence.getSetting<boolean>("privacy"),
+		]),
+		pages: Record<string, PresenceData> = {
+			"/about": { details: "Viewing About Page" },
+			"/gallery": { details: "Viewing Gallery Page" },
+			"/events": { details: "Browsing Events Page" },
+			"/newsletter": { details: "Browsing Newsletter" },
+			"/rules": { details: "Reading the rules" },
+			"/PrivacyPolicy": { details: "Reading Privacy Policy" },
+			"/Terms&Condition": { details: "Reading Terms&Condition" },
+			"/leaderboard": { details: "Viewing Leaderboard" },
+		};
+
+	for (const [path, data] of Object.entries(pages)) {
+		if (document.location.pathname.includes(path))
+			presenceData = { ...presenceData, ...data };
+	}
+
+	switch (document.location.host) {
+		case "nexonlogistics.com":
+			switch (true) {
+				case document.location.pathname.includes("/newsletter/"):
+					presenceData.details = "Reading a newsletter";
+					presenceData.state = document.querySelector<HTMLHeadingElement>(
+						"div.container-fluid > h1"
+					).textContent;
+					presenceData.buttons = [{ label: "Read Post", url: document.URL }];
+					break;
+				case document.location.pathname.includes("/convoy"):
+					presenceData.details = "Browsing Nexon Events";
+					presenceData.state =
+						document.querySelectorAll<HTMLHeadingElement>("h1")[1].textContent;
+					presenceData.buttons = [{ label: "View Event", url: document.URL }];
+					break;
+				case document.location.pathname.includes("/register"):
+					presenceData.details = "Browsing Nexon Applications";
+					presenceData.state =
+						document.querySelectorAll<HTMLHeadingElement>("h1")[1].textContent;
+					presenceData.buttons = [
+						{ label: "View Application", url: document.URL },
+					];
+					break;
+				case document.location.pathname.includes("/index"):
+					presenceData.details = "Viewing Nexon Homepage";
+					presenceData.state =
+						document.querySelector<HTMLHeadingElement>("h1").textContent;
```

Comment:
Based on [this conversation](https://github.com/PreMiD/Presences/pull/7044/commits/33181b3b01efc4cc0326a44a52caad810fa34efb#r1035057075)
```suggestion
						document.querySelectorAll<HTMLHeadingElement>("h1")[1].textContent;
```